Sources/AppStoreConnect/Networking/Pagination.swift

```diff
@@ -15,17 +15,21 @@ public struct PagedResponses<Response: Decodable & Sendable>: AsyncSequence, Asy
     let request: Request<Response>
     /// A reference to the API client.
     let client: AppStoreConnectClient
+    /// The retry strategy.
+    let retryStrategy: any RetryStrategy
```

Comment:
Seems a great choice to not expose as a generic it in PagedResponses 🎉 . Should we mention it?